projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -130,6 +130,15 @@ function wpcom_launchpad_get_task_definitions() {
 				return '/me/account';
 			},
 		),
+		'verify_domain_email'             => array(
+			'get_title'           => function () {
+				return __( 'Verify email address', 'jetpack-mu-wpcom' );
+			},
+			'is_visible_callback' => 'wpcom_launchpad_is_domain_email_unverified',
+			'get_calypso_path'    => function () {
+				return '/me/account';
+			},
```

Comment:
I'm thinking we should drop this and go with the ICANN verification logic: https://github.com/Automattic/wp-calypso/blob/49c6391c280b60c611e176315471e25a5b3e6f60/client/data/domains/use-domain-email-verfication.ts#L34-L38.

Reply:
I'm not sure if we can send an email after clicking this. I think sending them to `/domains/manage/all/:domain/edit/:site` would be better.

Reply:
I used `/domains/manage/:site` as it is easy to produce this URL and still contains all the relevant info, including allowing you to change the email address.